rw_tlv.m4

```diff
@@ -596,4 +536,4 @@ m4_define(['m4_define_plus'],
 // --------------------------
 
 
-m4_divert['']m4_dnl
+m4_divert['']m4_dnl
```

Comment:
End file w/ newline. M4 is picky. My pre-processing should add it, but let's add it to the file so pre-processing doesn't have to modify.